src/reducer/filter-reducer.js

```diff
@@ -0,0 +1,28 @@
+export const reducer = (state, action) => {
+    switch (action.type) {
+        
+        case "LOW_TO_HIGH":
+            return { ...state, sortBy: action.type };
+
+        case "HIGH_TO_LOW":
+            return { ...state, sortBy: action.type };
+
```

Comment:
you can give payload with type , as it will help in reducing the number of switch cases and will include same type of states together.
Just a suggestion.

Reply:
ok i will refactor  in future